app/views/shared/views/_alerts_list.html.haml

```diff
@@ -67,7 +67,10 @@
       %list-expanded-content
         .row
           .col-md-12
-            %a{"ng-href" => '{{$parent.$parent.item.sopLink}}', "target" => "_blank", "ng-if" => "$parent.item.sopLink"}
+            %a{"ng-href" => '{{$parent.$parent.item.sopLink}}',
+               "target" => "_blank",
+               "ng-if" => "$parent.item.sopLink",
+               :rel => 'noopener noreferrer'}
```

Comment:
why is there a mix of strings and symbols here?  Is that intentional?

Reply:
We prefer symbols, but angular attribute names use dashes in the name, so those have to be strings.